sdks/python/apache_beam/testing/benchmarks/cloudml/cloudml_benchmark_test.py

```diff
@@ -138,7 +138,7 @@ def test_cloud_ml_benchmark_criteo_fixed_workers_10GB(self):
     workflow.run(test_pipeline.get_full_options_as_args(**extra_opts))
     end_time = time.time()
 
-    metrics_table = 'cloudml_benchmark_criteo_10GB'
+    metrics_table = 'cloudml_benchmark_criteo_fixed_workers_10GB'
```

Comment:
can you remind please how many workers do we use?

Reply:
we use 50 workers. should i include the worker and machine type in the table name?

Reply:
I think 50 workers may be too much for 10 GB. It might be that the benchmark will not be very sensitive if it completes too fast, and most of the runtime time is dominated by worker startup/shutdown

Reply:
I would probably select around 5 workers. o